### ISSUE TITLE
r7-8845hs

### ISSUE THREAD

**Issue by Dllzzz** [JamesCJ60/Universal-x86-Tuning-Utility#258]
I used this software to undervolt my laptop equipped with an R7-8845HS, but I couldn't confirm if it took effect. Can you tell me if the 8845HS is compatible with this software?

**Comment by neoskynet** [comment]
It also doesn’t work for me for r7-8845HS on an acer nitro 16 an16-42-r287 laptop. Does anyone know how to reduce the voltage on this processor?
maybe there is other software?

**Comment by X4ero26** [comment]
> He pasado dias tratando de limitar el voltaje y consumo del R7-8845HS ya que en laptop lo maximo que deberia consumir es 45 w pero a mi me consume 55w y su voltaje es alto y la temperatura me ha llegado a los 100 grados, no encuentro ninguna solucion, ni con BIOS ya que mi laptop es HP y HP bloquea estas funciones y ningun software de AMD o de terceros es compatible con este procesador

Encontraste una solución? Me pasa algo parecido

**Comment by JamesCJ60** [comment]
Hi, the Ryzen 7 8845HS does not support undervolting via Curve Optimiser. For product segmentation purposes on AMD’s end, only Ryzen 9 and/or HX APUs/CPUs have the functionality intact after the 4000 series.

**Comment by Sage222** [comment]
yeah i have the same CPU and from my observations of voltage under load with the latest version 26.2.0b4 the Curve Optimiser doesnt do anything.   Havent tried the iGPU yet..
